src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -175,35 +191,289 @@ type SingleBlastJobResultProps = {
+          <BlastSequenceAlignment
+            alignment={sequenceAlignmentData[rowId]}
+            blastDatabase={blastDatabase}
+          />
+        </div>
+      );
+      setExpandedContent({
+        [rowId]: aligment
+      });
+    }
+  };
+
+  hitsTableColumns[3].renderer = (params: TableCellRendererParams) => {
+    return (
+      <ShowHideColumn
+        onExpanded={onExpanded}
+        rowId={params.rowId}
+        isExpanded={!!expandedContent?.[params.rowId]}
+      />
+    );
+  };
+
   if (!data) {
     return null;
   }
 
   const alignmentsCount = countAlignments(data.result);
 
   return (
-    <div className={styles.resultsSummaryRow}>
-      <div className={styles.hitLabel}>
-        <span>{alignmentsCount} </span>
-        <span className={styles.label}>
-          {`${pluralise('hit', alignmentsCount)}`}
-        </span>
-      </div>
-      <div className={styles.summaryPlot}>
-        <BlastHitsDiagram job={data.result} width={diagramWidth} />
-      </div>
-      <div className={styles.speciesInfo}>
-        {speciesInfo.common_name && <span>{speciesInfo.common_name}</span>}
-        <span>{speciesInfo.scientific_name}</span>
-        <span>{speciesInfo.assembly_name}</span>
+    <>
+      <div className={styles.resultsSummaryRow}>
+        <div className={styles.hitLabel}>
+          <span>{alignmentsCount} </span>
+          <span className={styles.label}>
+            {`${pluralise('hit', alignmentsCount)}`}
+          </span>
+        </div>
+        <div className={styles.summaryPlot}>
+          <BlastHitsDiagram job={data.result} width={diagramWidth} />
+        </div>
+        <div className={styles.speciesInfo}>
+          {speciesInfo.common_name && <span>{speciesInfo.common_name}</span>}
+          <span>{speciesInfo.scientific_name}</span>
+          <span>{speciesInfo.assembly_name}</span>
+          <div className={styles.showHideWrapper}>
+            <ShowHide
+              isExpanded={isExpanded}
+              onClick={() => setExpanded(!isExpanded)}
+            ></ShowHide>
+          </div>
+        </div>
+
+        {isExpanded && (
+          <div className={styles.tableWrapper}>
+            <DataTable
+              state={tableState}
+              onStateChange={setTableState}
+              columns={hitsTableColumns}
+              theme="dark"
+              className={styles.hitsTable}
+              expandedContent={expandedContent}
+              disabledActions={[TableAction.FILTERS]}
+              uniqueColumnId="id"
+            />
+          </div>
+        )}
       </div>
-    </div>
+    </>
+  );
+};
+
+const ShowHideColumn = (props: {
+  isExpanded: boolean;
+  onExpanded: (isExpanded: boolean, rowId: string) => void;
+  rowId: string;
+}) => {
+  const onExpanded = () => {
+    props.onExpanded(!props.isExpanded, props.rowId);
+  };
+
+  return (
+    <ShowHide
+      label="view alignment"
+      isExpanded={props.isExpanded}
+      onClick={onExpanded}
+    />
```

Comment:
Can't hide the alignment:

https://user-images.githubusercontent.com/6834224/188394306-9d861d72-36b8-4577-81e3-aeca2a144d0b.mov

Reply:
This should be fixed now